addons/mail/static/src/models/thread/thread.js

```diff
@@ -1711,10 +1824,27 @@ function factory(dependencies) {
                 'serverFoldState',
             ],
         }),
+        /**
+         * All messages ordered like they are displayed.
+         */
         orderedMessages: many2many('mail.message', {
             compute: '_computeOrderedMessages',
             dependencies: ['messages'],
         }),
+        /**
+         * Is transient trait for all messages ordered like they are displayed.
```

Comment:
I wonder if we shouldn't "mark" all those field with a tag or a PR or something so that when we allow the dot notation to be used there we find them easily and remove those now useless compute dependency field ?

Reply:
Yes that's pretty much what "Serves as compute dependency." does, if all have that comment it's easy to find them.